Feature/adding model

abhineetgupta opened the pull request:
Added model code file and readme. Both to be reviewed, however, functions are mostly the same as already reviewed.

lilsummer commented on README.md:
````diff
@@ -1 +1,103 @@
+
+### Dependencies
+```
+numpy
````
is `logging` a dependency? since it is used in the py file

abhineetgupta replied:
It's available as a standard library in python, like `math`.

lilsummer commented on covid_workplace_model.py:
```diff
@@ -0,0 +1,461 @@
+    """
+    virion_additional_decay_factor = 0.62
+
+    speaking_percentage_benchmark_sar = 0.25
```
A minor comment: it might be helpful to add a comment explaining how we got the `benchmark` values

abhineetgupta replied:
Good point. Will do.

lilsummer commented on covid_workplace_model.py:
```diff
@@ -0,0 +1,461 @@
+        previous_contacts = {
+            key: np.empty(0, dtype=np.int32) for key in infected_index_gen1
+        }
+        infected_index_contacts = np.empty(0, dtype=np.int32)
```
suggest add a sentence to explain the intent of the following for loop

abhineetgupta replied:
Added a sentence.

lilsummer commented on covid_workplace_model.py:
```diff
@@ -0,0 +1,461 @@
+                num_contacts_for_day = num_contacts_daily[
+                    : running_sum_infected_gen1[day], day
+                ]
+
```
suggest add a sentence to explain the intent of the following for loop

abhineetgupta replied:
Added.

lilsummer commented on covid_workplace_model.py:
```diff
@@ -0,0 +1,461 @@
+        type=float,
+        metavar="[0., 1.]",
+        required=True,
+        help="daily new cases per population",
```
suggestion on changing this to per 100k population, right?

abhineetgupta replied:
No, because units are per population. So if cases are 1000 per 100k population, value here is 0.01.

lilsummer reviewed (commented):
Great modification, thanks for adding all the documentation, reference, docstring, and argparse. I left some minor comments

abhineetgupta commented:
> Great modification, thanks for adding all the documentation, reference, docstring, and argparse. I left some minor comments

@lilsummer Thanks for the comprehensive review. I have addressed all comments.

lilsummer reviewed (approved):
LGTM
